controllers/articles.js

```diff
@@ -29,15 +29,59 @@ router.get('/new', (req, res) => {
 })
 
 // GET /articles/:id - display a specific post and its author
+// router.get('/:id', (req, res) => {
+//   db.article.findOne({
+//     where: { id: req.params.id },
+//     include: [db.author]
+//   })
+//   .then((article) => {
+//     if (!article) throw Error()
+//     // console.log(article.author)
+//     res.render('articles/show', { article: article })
+//   })
+//   .catch((error) => {
+//     console.log(error)
+//     res.status(400).render('main/404')
+//   })
+// })
+
+// GET route /article/id
 router.get('/:id', (req, res) => {
   db.article.findOne({
     where: { id: req.params.id },
     include: [db.author]
   })
   .then((article) => {
     if (!article) throw Error()
-    console.log(article.author)
-    res.render('articles/show', { article: article })
+    db.comment.findAll({
+      where: {articleId: req.params.id}
+    })
```

Comment:
This is a way to find the comments that have a particular articleId.  An easier way is to add the comments that are associated with the article when you find that article higher up in this route.  If you see where you include the db.author on line 52, you can also include the db.comment to grab all of the associated comments.  Just want to note this for your own knowledge!  Ex for line 52:

include: [db.author, db.comment]